basculechecks/metricvalidator.go

```diff
@@ -32,7 +32,12 @@ var defaultLogger = log.NewNopLogger()
 // authorized given a bascule.Authentication object.  If it's not authorized,
 // a reason and error are given for logging and metrics.
 type CapabilitiesChecker interface {
-	Check(auth bascule.Authentication) (string, error)
+	Check(auth bascule.Authentication, vals ParsedValues) (string, error)
+}
+
+type ParsedValues struct {
+	Endpoint string
```

Comment:
It might be worth adding a comment to explain the meaning we it give and how it's generated. Something like:
```
// Endpoint represents a bucket of URLs to be validated. An endpoint could be the regex string (i.e. `.*/api/v1/account/.*`) that matched the URL of a particular HTTP request whose capabilities need to be validated. 
```
Same with partner